onnxruntime/python/onnxruntime_inference_collection.py

```diff
@@ -627,6 +690,12 @@ def numpy(self):
         Use accessors to gain a reference to non-Tensor objects such as SparseTensor
         '''
         return self._ortvalue.numpy()
+    
+    def update_inplace(self, np_arr):
+        '''
+        Update the cuda OrtValue in place with a new Numpy array.
```

Comment:
Is CUDA OtrValue relevant in the comment ? Looking at the implementation, it seems like the inplace update can happen even on CPU / ROCM ?

Reply:
Remove `cuda` here.

Reply:
Maybe also add a line stating the numpy contents are copied over to the device memory backing the OrtValue ?

Reply:
Is this change reflected in the latest update ?

Reply:
It is updated to be as below:

```
Update the OrtValue in place with a new Numpy array. The numpy contents
are copied over to the device memory backing the OrtValue.
```